spec/javascripts/NewService/ServiceDiscoveryForm.spec.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Enzyme, {shallow, mount} from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+
+import {ServiceDiscoveryForm} from 'NewService'
+
+Enzyme.configure({adapter: new Adapter()})
+
+const Props = {
```

Comment:
PascalCase is reserved for classes, component, types... Use better camel or UPPER.